e2e-tests/steps/import/import-steps.js

```diff
@@ -272,7 +262,7 @@ class ImportSteps {
     }
 
     static importFile(index) {
-        this.getResource(index).find('.import-resource-action-import-btn').click();
+        this.getResource(index).find('.import-resource-action-import-btn').click({force: true});
```

Comment:
Why with the force?

Reply:
changed to .invoke('show').trigger('click') to simulate hover+click